src/auth/digest/UserRequest.cc

```diff
@@ -357,10 +357,13 @@ Auth::Digest::UserRequest::HandleReply(void *data, const Helper::Reply &reply)
 
     case Helper::TT:
         debugs(29, DBG_IMPORTANT, "ERROR: Digest auth does not support the result code received. Using the wrong helper program? received: " << reply);
+    /* [[fallthrough]] */
     // fall through to next case. Handle this as an ERR response.
 
     case Helper::TimedOut:
+    /* [[fallthrough]] */
```

Comment:
I hope this comment is not needed. The code is more readable without it and there are many other sequential `case:...` lines in the code that you have not modified. If this comment is not needed indeed, please do not add it.

```suggestion
```

Reply:
Not sure, but let's try